examples/fewshot_protonet/README.md

```diff
@@ -0,0 +1,102 @@
+# Prototypical Networks for Few-shot Learning
+
+### 1. Description
+
+This demo implements [Prototypical Networks for Few-shot Learning](https://github.com/jakesnell/prototypical-networks) within the `PyKale` framework. The following instructions are about how to use it.
+
+ProtoNet is for few-shot learning under $N$-Way $K$-Shot settings:
+
+$N$-way: It means the number of classes or categories in one epoch in evaluation. For instance, in a 5-way scenario, data from five classes is fed into the model in an epoch.
+
+$K$-shot: This is the number of examples (referred to as "shots") from each class in training and evaluation. A 1-shot learning task indicates only one example per class, while a 3-shot task has three examples per class.
```

Comment:
Why $K$ in training AND evaluation? Do we have to have the same number of examples for training and evaluation? Confusing.

Reply:
Will explain it face to face.